db/models.py

```diff
@@ -50,3 +52,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to='User',
+                             on_delete=models.CASCADE,
+                             related_name="orders")
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
```

Comment:
The `class Meta` is implemented before some attributes are defined. Remember that it's a good practice to define all attributes before implementing `class Meta`.